serverless/event_workflows/serverless-event-delivery.adoc

```diff
@@ -0,0 +1,36 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-event-delivery"]
+= Configuring event delivery parameters
+include::modules/common-attributes.adoc[]
+:context: serverless-event-delivery
```

Comment:
I think the status code docs fit in this page, you can have a paragraph like _Eventing components behaviours_ where we state the various guarantees, the status codes etc

Reply:
Added for Kafka channel, also added some notes / questions / TODOs for other components. If you have info for these that would be very helpful?